webapp-django/crashstats/settings/base.py

```diff
@@ -177,18 +176,33 @@ def path(*dirs):
 # Always generate a CSRF token for anonymous users.
 ANON_ALWAYS = True
 
-LOG_LEVEL = logging.INFO
+LOGGING_LEVEL = config('LOGGING_LEVEL', 'INFO')
```

Comment:
This is nice because then it's easier to change in `my.env` when you're doing things.